app/models/manageiq/providers/amazon/inventory/persister/target_collection.rb

```diff
@@ -24,6 +24,10 @@ def initialize_inventory_collections
 
   private
 
+  def references(collection)
```

Comment:
is this a generic method?
Can you get define it in somewhere like `ManagerRefresh::Inventory::Collector` or highter?